lib/FusionInventory/Agent/Tools/Win32.pm

```diff
@@ -421,26 +386,9 @@ sub getInterfaces {
                     DESCRIPTION => $configuration->{DESCRIPTION},
                     STATUS      => $configuration->{STATUS},
                     MTU         => $configuration->{MTU},
-                    dns         => $configuration->{dns},
+                    dns         => $configuration->{dns}
                 };
 
-                if ($params{additionalProperties}
-                    && $params{additionalProperties}->{NetWorkAdapterConfiguration}) {
-                    for my $prop (@{$params{additionalProperties}->{NetWorkAdapterConfiguration}}) {
-                        if ($configuration->{$prop}) {
-                            $interface->{$prop} = $configuration->{$prop};
-                        }
-                    }
-                }
-                if ($params{additionalProperties}
-                    && $params{additionalProperties}->{NetWorkAdapter}) {
-                    for my $prop (@{$params{additionalProperties}->{NetWorkAdapter}}) {
-                        if ($object->{$prop}) {
-                            $interface->{$prop} = $object->{$prop};
-                        }
-                    }
-                }
-
```

Comment:
I have no clue about the usefulness of this additionalProperties function parameter, but this simplification doesn't seem related to the global commit purpose, as defined by the commit message.

Reply:
Yes, I agree. I'll update the commit message during the `squash and commit` with:
> avoid to handle additionalProperties parameters in win32 getInterfaces() API
> for just one module case when we can still simply merge them as not so heavy
> to carry
Is that okay for you ?